src/components/resortPage/__tests__/SpecificResortPage.test.js

```diff
@@ -8,7 +9,7 @@ describe('SpecificResortPage', () => {
       },
     },
     getSpecificResort: () => { },
-    resort: fixture.resort,
+    resort: resortId1,
```

Comment:
You could use resort: resorts[0] so there is no need for adding separate fixtures unless they contain some special use cases, like for example: resort without piste, or something like that.